backend/db/initialize.py

```diff
@@ -155,6 +155,14 @@ def initialize(
 
         make_derived_tables_and_more(con, schema, local=local)  # , start_step=30)
 
+        # Make update audit tables
+        # - these tables are to store history of updates made to 'core' tables; when they were made, not by who
+        audit_update_tables = ['code_sets']
```

Comment:
## Audit tables for updates & deletes
I set up a way to do this, so we can see the history of updates and possibly undo changes.

Added to `initialize()`.